src/Controls/tests/Core.UnitTests/ElementTests.cs

```diff
@@ -313,5 +313,62 @@ public void ChildRemoved()
 			root.Children.Remove(child);
 			Assert.True(removed);
 		}
+
+		[Fact]
+		public void RealParent_ReturnsNullAfterParentGarbageCollected()
```

Comment:
Can include another test?
- RealParent_ReturnsNull_WhenNoParentSet

and maybe something like:
- RealParent_IsThreadSafe_WhenAccessedConcurrently